src/app/examples/card-example/card-example.component.tns.html

```diff
@@ -1 +1,15 @@
-<Button text="card-example works!" class="btn btn-primary"></Button>
\ No newline at end of file
+<FlexboxLayout width="300">
```

Comment:
Flyt til SCSS? Eller er det fordi vi bruger det programmatisk?
```suggestion
<FlexboxLayout>
```

Reply:
Det er egentlig blot til at teste skalering, så det skal blot fjernes